pkg/controller/registry/reconciler/grpc.go

```diff
@@ -226,7 +226,7 @@ func (c *GrpcRegistryReconciler) ensurePod(source grpcCatalogSourceDecorator, sa
 			return nil
 		}
 		for _, p := range currentLivePods {
-			if err := c.OpClient.KubernetesInterface().CoreV1().Pods(source.GetNamespace()).Delete(context.TODO(), p.GetName(), *metav1.NewDeleteOptions(0)); err != nil {
+			if err := c.OpClient.KubernetesInterface().CoreV1().Pods(source.GetNamespace()).Delete(context.TODO(), p.GetName(), *metav1.NewDeleteOptions(1)); err != nil {
```

Comment:
Probably easiest if I also add a package constant set to 1 and just reference that variable in `NewDeleteOptions(...)` instead of hardcoding the value of 1. I need to double-check what the default grace period seconds are, and whether we can just pass an empty metav1.DeleteOptions object instead of explicitly setting the grace period every time.

Reply:
I'm partial to a package scoped constant _that is also_ the zero value for `metav1.DeleteOptions`: `metav1.DeleteOptions{}`.

According to the [type docs](https://pkg.go.dev/k8s.io/apimachinery@v0.21.0-beta.1/pkg/apis/meta/v1#DeleteOptions), as long as the `GracePeriodSeconds` field is `nil`:

> ... the default grace period for the specified type will be used.